src/classes/Auth/Auth.ts

```diff
@@ -70,14 +70,22 @@ export class Auth extends PassageBase {
      * Create a Magic Link for your app.
      *
      * @param {CreateMagicLinkArgs} args options for creating a MagicLink.
+     * @param {MagicLinkOptions} options options for creating a MagicLink.
      * @return {Promise<MagicLink>} Passage MagicLink object
      */
-    public async createMagicLink(args: CreateMagicLinkArgs): Promise<MagicLink> {
+    public async createMagicLink(args: CreateMagicLinkArgs, options?: MagicLinkOptions): Promise<MagicLink> {
         try {
             const magicLinksApi = new MagicLinksApi(this.config.apiConfiguration);
+            const { language, magicLinkPath, redirectUrl, ttl } = options || {};
```

Comment:
nitpick: i think in this case the `??` operator is [preferred](https://typescript-eslint.io/rules/prefer-nullish-coalescing/) since it checks `null`/`undefined` specifically while the `||` operator checks for `falsy`

Reply:
You're right, thanks Copilot, should've been paying more attention.